### DISCUSSION TITLE
Is github down today?

### DISCUSSION THREAD

**Discussion by Annesha003** [community/community#51113] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I tried creating a new repository , also tried adding to my readme it just isn't working.It is showing me error 500.

**Comment by GyulyVGC** [comment]
It's a problem on GitHub side: see [here](https://www.githubstatus.com) @Annesha003 
We just have to wait and hopefully it'll be solved soon.
Don't forget to mark this comment as answer so that other people can view this discussion as solved! 

![Screenshot 2023-03-27 at 15 05 15](https://user-images.githubusercontent.com/100347457/227947004-184645b4-1590-40d8-afe0-825a878b6934.png)

**Comment by PajaritoMoyqi** [comment]
Things are better now than before.

**Comment by LoveForInclusion** [comment]
So and so, it finally created a repo for me but i still can't push.

**Comment by Annesha003** [comment]
I am still not able to commit the changes in my readMe .

**Comment by dawidzawada** [comment]
Currently I can't see any of my pushed commits in PR but I can download branch and changes are there - creating new PR helped

**Comment by gkurz** [comment]
I having a similar issue : if I try to push anything to a cloned repo on GH, it fails.

```
$ git remote get-url origin
https://github.com/kata-containers/kata-containers
$ git remote get-url --push origin
git@github.com:gkurz/kata-containers.git
$ git checkout origin/main -b push-test
branch 'push-test' set up to track 'origin/main'.
Switched to a new branch 'push-test'
$ git push -f origin push-test
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
$
```
